src/app/routes/keycloakSignInRoutes.js

```diff
@@ -0,0 +1,84 @@
+const bodyParser = require('body-parser');
+const envHelper = require('../helpers/environmentVariablesHelper.js');
+const request = require('request-promise');
+var qs = require('querystring');
+var DOMParser = require('xmldom').DOMParser;
+const parser = new DOMParser();
+
+module.exports = (app) => {
+  app.post('/keycloak/login', bodyParser.json(), async (req, res) => {
+    const userName = req.body.emailId;
```

Comment:
@vpPavithra 
Add necessary logs (logger statement) for each API endpoint and activities